flux-syntax/src/surface.rs

```diff
@@ -147,6 +152,7 @@ pub struct Expr {
 
 #[derive(Debug, Clone)]
 pub enum ExprKind {
+    Const(DefId),
```

Comment:
could it be that we don't need this in surface because the resolving happens during desugaring?

Reply:
Yup!